lib/core/scheduler.rb

```diff
@@ -59,37 +59,47 @@ def shutdown
     private
 
     def sync_triggered?(connector_settings)
+      unless Connectors::REGISTRY.registered?(connector_settings.service_type)
```

Comment:
Crawler uses the same scheduler (see [scheduler.rb](https://github.com/elastic/connectors-ruby/blob/main/lib/connectors/crawler/scheduler.rb)) so this is a breaking change. 

I think Crawler connector should be ignored in ConnectorsSettings level.

Reply:
Oh wow, I missed it fully :/ Great catch!

Reply:
Hi @vidok , I'm a bit confused here. Do you expect `connectos-ruby` to run sync job for crawler? I assume the answer is no, because don't see `connctor` class in [crawler](https://github.com/elastic/connectors-ruby/tree/main/lib/connectors/crawler).

If this is true, this PR will make `connectors-ruby` ignore crawler connecters because `crawler` is not one of the registered connectors:

https://github.com/elastic/connectors-ruby/blob/afb58a6ee3e39de6cbe1bbf655182ea57faddeeb/lib/connectors/registry.rb#L42-L51

Reply:
Oh, I got it. This is used as a utility gem in ent-search. Great catch @vidok

Reply:
Hi @vidok , will this work? https://github.com/elastic/connectors-ruby/pull/327/commits/e61f78be3ad5aaaed91b006c809abd8a327ad543

Reply:
Yes, I think this should work, however it requires some testing.